src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: $product-card-width;
+  background-color: #fff;
+  border: 1px solid $main-border-color;
+  border-radius: 5px;
+  transition: transform 0.3s;
+
+  &__product-image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+    transition: color 0.3s;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-text-color;
+    margin: 0 0 16px;
+  }
+
+  &__product-rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin: 0 0 24px;
+
+    .reviews-count {
+      margin: 0;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__product-price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 0 16px;
+
+    .price-title {
+      margin: 0;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $secondary-text-color;
+    }
+
+    .price-numbers {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    height: 40px;
+
+    background-color: $blue-brand-color;
+    border: 1px solid $blue-brand-color;
+    border-radius: 5px;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+    transition: all 0.3s;
+
+    &:hover {
+      background-color: #fff;
+      color: $blue-brand-color;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
It's a good practice to avoid using transform scale on hover for large elements like cards because it can cause layout shifts and affect performance. Consider using a different effect, such as a box-shadow or a border change, to indicate hover state without causing layout shifts.